src/main/java/com/springboot/thymeleafdemo/dao/EmployeeDAOImpl.java

```diff
@@ -64,7 +64,17 @@ public void save(Employee theEmployee) {
 		Session currentSession = entityManager.unwrap(Session.class);
 
 		// save employee
-		currentSession.saveOrUpdate(theEmployee);
+		currentSession.save(theEmployee);
```

Comment:
instead of `theEmployee`, u can use `employee`
If you have time, please change or else leave it and remember for the next time

Reply:
Yeah Sure. Will refactor those.